core/azure-core/src/main/java/com/azure/core/polling/PollResponse.java

```diff
@@ -0,0 +1,128 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.core.polling;
+
+import java.time.Duration;
+import java.util.Map;
+import java.util.Objects;
+
+/**
+ * Container of information related to polling and poll response. It will hold current polling status {@link OperationStatus} , T value,
+ * retry after duration and various configuration properties.
+ *
+ * <p><strong>Container of PollResponse and related information.</strong></p>
+ * @param <T> type of poll response value
```

Comment:
Use `@see` links to link the relevant classes.

Reply:
resolved